Scripts/create_pods_and_versions_index.rb

```diff
@@ -2,47 +2,150 @@
 # frozen_string_literal: true
 
 require 'cocoapods'
+require 'parallel'
 
-source = Pod::Source.new('.')
+def parallel?
```

Comment:
Is there any reason for this configuration?

Reply:
wanted to make it easy to turn off parallelism if it caused issues, though I can see how that might not be worthwhile